### ISSUE TITLE
can't start wyze bridge

### ISSUE THREAD

**Issue by ibanez78** [mrlt8/docker-wyze-bridge#944]
just today my wye bridge stopped working, when I try to start I get the following:

docker run \
-e WYZE_EMAIL=*email* \
-e WYZE_PASSWORD=*pass* \
-p 1935:1935 -p 8554:8554 -p 8888:8888 -p 5000:5000 \
mrlt8/wyze-bridge:latest

I get the following error
* Serving Flask app 'frontend'
[WyzeBridge] 🔍 Could not find local cache for 'auth'
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
* Debug mode: off
[WyzeBridge] WARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.
* Running on all addresses (0.0.0.0)
* Running on http://127.0.0.1:5000/
* Running on http://172.17.0.2:5000/
[WyzeBridge] Press CTRL+C to quit
[WyzeBridge] ⚠️ 403 Client Error: Forbidden for url: https://auth-prod.api.wyze.com/user/login
[WyzeBridge] 🔍 Could not find local cache for 'user'
[WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
[WyzeBridge] 🔍 Could not find local cache for 'auth'
[WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...

after reading some posts 2FA was mentioned so I setup an API key and am trying to add that to my command but still getting the same error

docker run \
  -e WYZE_EMAIL=*email* \
  -e WYZE_PASSWORD=*pass* \
  -e API_ID=*api id* \
  -e API_KEY=*api key* \
  -p 1935:1935 -p 8554:8554 -p 8888:8888 -p 5000:5000 \
  mrlt8/wyze-bridge:latest

any ideas?
thanks for any help

**Comment by mrlt8** [comment]
Wyze made some changes to their auth API earlier this year, and you need to update your bridge to v2.3+ which will use the newer endpoints.

**Comment by ibanez78** [comment]
thanks for the info, doesn't "mrlt8/wyze-bridge:latest" take the latest version ( I assume 2.3 or higher)?

**Comment by mrlt8** [comment]
Yes, but you probably need to pull the latest image with something like this:

```
docker-compose pull # Pull new image
docker-compose up -d # Restart container in detached mode
docker image prune # Remove old images

```

**Comment by ibanez78** [comment]
thanks, just ran "docker image pull mrlt8/wyze-bridge" and it works now, thanks again. More questions to follow...
